http/src/request/guild/sticker/mod.rs

```diff
@@ -0,0 +1,91 @@
+mod create_guild_sticker;
+mod delete_guild_sticker;
+mod get_guild_sticker;
+mod get_guild_stickers;
+mod update_guild_sticker;
+
+pub use self::{
+    create_guild_sticker::CreateGuildSticker, delete_guild_sticker::DeleteGuildSticker,
+    get_guild_sticker::GetGuildSticker, get_guild_stickers::GetGuildStickers,
+    update_guild_sticker::UpdateGuildSticker,
+};
+
+use std::{
+    error::Error,
+    fmt::{Display, Formatter, Result as FmtResult},
+};
+
+/// Error created if validation of a sticker field fails.
+#[derive(Debug)]
+pub struct StickerValidationError {
```

Comment:
Shouldn't this be a part of `request/validate.rs`?

Reply:
That file is subject to a full rework at some point anyway, so it shouldn't matter too much right now